src/main/webapp/js/components/app.jsx

```diff
@@ -41,7 +41,7 @@ class App extends Component {
     return (
       <Router>
         <Switch>
-          <Route path="/login" component={Login} />
+          {!this.props.authenticated ? <Route path="/" component={Login} /> : null}
```

Comment:
you don't need to use ternary operator here, it can be `!this.props.authenticated && <Route `,
also this should be after the the register and accessDenied